src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costRent = days * 40;
+  const discountFor3Days = 20;
+  const discountFor7Days = 50;
+
+  if (days >= 7) {
+    return costRent - discountFor7Days;
+  } else if (days >= 3) {
```

Comment:
If you write `return` inside `if` block, the next `else` is useless.

With `return` structure should look like this:

```
if (condition1) {
  return x;
}

if (condition2) {
  return y
}

return z;
```